src/Marten/MartenRegistry.cs

```diff
@@ -347,6 +347,24 @@ public DocumentMappingExpression<T> FullTextIndex(params Expression<Func<T, obje
                 return this;
             }
 
+            public DocumentMappingExpression<T> NgramIndex(Action<NgramIndex> configure)
```

Comment:
@apereiratl, could you add comments for the public API, to help people using intellisense?

Reply:
Yep. I can push a fix for this.